recipes/beast2/meta.yaml

```diff
@@ -15,16 +15,18 @@ source:
     - citation.patch
 
 build:
-  number: 2
+  number: 3
   noarch: generic
+  run_exports:
+    - {{ pin_subpackage("beast2", max_pin="x.x") }}
 
 requirements:
   build:
     - gettext
     - openjdk
     - ant
   host:
-    - openjdk 8.0* zulu8*
+    - openjdk 11.0* zulu8*
```

Comment:
change to `openjdk 11.0*`, without `zuli8*`

Reply:
ok